ChemistryLib/PhreeqcKernel.cpp

```diff
@@ -166,6 +212,12 @@ void PhreeqcKernel::setAqueousSolutions(
                 // Set pH value by hydrogen concentration.
                 double const pH = -std::log10(concentration);
                 aqueous_solution.Set_ph(pH);
+
+                if (auto hydrogen = components.find("H(1)");
+                    hydrogen != components.end())
+                {
+                    hydrogen->second.Set_input_conc(pH);
+                }
```

Comment:
Sorry for you. I forgot we have an old compiler setup here; This feature is only c++17. Old style is:
```suggestion
                {
                    auto hydrogen = components.find("H(1)");
                    if (hydrogen != components.end())
                    {
                        hydrogen->second.Set_input_conc(pH);
                    }
                }
```